test/sequential/test-inspector-open.js

```diff
@@ -80,7 +80,7 @@ function reopenAfterClose(msg) {
 }
 
 function ping(port, callback) {
-  net.connect(port)
+  net.connect(port, '127.0.0.1')
```

Comment:
The inspector should "just work" with the default IMHO, the fact that it defaults to IPv4 goes a bit against that. I think it comes from here: https://github.com/nodejs/node/blob/f392ac0bbe3f3757872b9c7cdc785928034b62e9/src/node_options.h#L85
Not that it has anything to do with this PR, I just wanted to point it out.